.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can this be removed if it is not needed?

Reply:
We should add `.vscode/` to gitignore.